### ISSUE TITLE
1B mixture and validation data

### ISSUE THREAD

**Issue by thompsonb** [sail-sg/regmix#9]
I'm trying to find the data used to create Table 2, and I can see both the mixtures and the validation loss for the 1M and 60M results in the data/ directory. Could you point me to the same files (mixtures used and model validation losses) for the 1B models trained on 25B tokens? Thanks!

**Comment by SivilTaram** [comment]
@thompsonb Thanks for your interest! I will upload the corresponding files of 1B models later.

**Comment by thompsonb** [comment]
Thanks @SivilTaram! Quick question - why are the 1B models only evaluated on 7 subsets of the pile instead of the 13 subsets used for the 1M and 60M models?

**Comment by SivilTaram** [comment]
@thompsonb Hello! The answer is simple: 1M and 60M losses are obtained from wandb, and the 1B loss is obtained from our later evaluation on the evaluation set. Sorry for any potential confusion! If you want more evaluation results, we can update the file later!

**Comment by thompsonb** [comment]
Yes, if it's easy to run inference on the other validation sets that would be great. In the meantime I'll use the 7 subsets you shared. Thanks again!

**Comment by SivilTaram** [comment]
@thompsonb Sorry for the late update (have been busy with ICLR submissions). We have updated the corresponding file with losses on 13 domains, thanks!

**Comment by thompsonb** [comment]
Perfect, thanks!
